### ISSUE TITLE
Rem mixin output duplicate values

### ISSUE THREAD

**Issue by imjared** [nclud/sketchbook#32]
```
#post-footer {
  padding-left: 18%;
  border-top: 1px solid rgba($dark-blue,0.35);
  @include rem(margin,4rem 0);
  @include rem(padding-top,2rem);
  ...
}
```

compiles to

```
#post-footer {
  padding-left: 18%;
  border-top: 1px solid rgba(15, 39, 51, 0.35);
  margin: 4rem 0;
  margin:  4rem 0;
  padding-top: 2rem;
  padding-top:  2rem; }
```

wut

**Comment by kyleconrad** [comment]
Weird. It's supposed to take REM and spit out a PX fallback, this is the first time I've ever had it just come back with REM as well instead of PX.

**Comment by imjared** [comment]
This seems to break the build. Did you verify before merging? May be an issue with something I have locally.

```
Running "sass:server" (sass) task
WARNING:  is an unknown length unit. Returning original value.
     /Users/jaredlaser/work/sketchbook/app/_scss/includes/_rem.scss:48, in function `convert-length`
     /Users/jaredlaser/work/sketchbook/app/_scss/includes/_rem.scss:103, in mixin `rem`
     /Users/jaredlaser/work/sketchbook/app/_scss/base/_sections.scss:63

WARNING:  is an unknown length unit. Returning original value.
     /Users/jaredlaser/work/sketchbook/app/_scss/includes/_rem.scss:48, in function `convert-length`
     /Users/jaredlaser/work/sketchbook/app/_scss/includes/_rem.scss:103, in mixin `rem`
     /Users/jaredlaser/work/sketchbook/app/_scss/base/_sections.scss:63

Warning: /Users/jaredlaser/work/sketchbook/app/_scss/includes/_rem.scss:97: error: argument `$number` of `unitless($number)` must be a number
Backtrace:
    /Users/jaredlaser/work/sketchbook/app/_scss/includes/_rem.scss:97, in function `unitless`
    /Users/jaredlaser/work/sketchbook/app/_scss/includes/_rem.scss:97, in mixin `rem`
    /Users/jaredlaser/work/sketchbook/app/_scss/_front.scss:31
 Use --force to continue.

Aborted due to warnings.
```

It seems to choke on `auto` values. I'd suggest we keep the previous mixin, comment out a line, and call it a day. I'm not sure we really need a px fallback since the site doesn't work in old browsers.

**Comment by imjared** [comment]
Closed by 61baf1577f741b2dff2dae961e21c355a8c1d0f2
